susemanager/src/mgr_bootstrap_data.py

```diff
@@ -103,11 +103,11 @@
 ]
 
 RES7 = [
-    "logrotate",
+    "logrotate*",
```

Comment:
logrotate is not an optional dependency for the Bundle, see e.g. https://build.suse.de/projects/SUSE:Maintenance:35074/packages/venv-salt-minion.SUSE_RES-7_Update_Products_SaltBundle_Update/repositories/SUSE_RES-7_Update_Products_SaltBundle_Update/binaries/x86_64/venv-salt-minion-3006.0-2.58.1.x86_64.rpm

Reply:
yes, but RES7 LTSS repo does not provide that package. It result always in build errors.
As I think logrotate is in 99% of all the cases already installed, I would vote for this shortcut.
It only effect RES7 which is LTSS. By the rules no, new installation should happen for LTSS versions.

Reply:
I have changed it and now we have logrotate only optional for `SLL7-OL-LTSS-x86_64` where the default channel does not have this package.